test/contracts/fail/10_BadVault.sol

```diff
@@ -166,39 +175,4 @@ contract BadVaultTest is Test {
         // console2.log("attacker final balance", address(attacker).balance);
         assert(attacker.balance <= STARTING_BALANCE);
     }
-
```

Comment:
why is the test being removed?

Reply:
No idea. I didn't do it :D I can add it back.

Reply:
OK, added back the test!